src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubTriggerInputBindingStrategy.cs

```diff
@@ -54,7 +42,26 @@ public EventData[] BindMultiple(EventHubTriggerInput value, ValueBindingContext
             return value.Events;
         }
 
-        // GEt runtime instance of binding contract 
+        // Get the static binding contract
+        //  - gets augmented 
+        public Dictionary<string, Type> GetStaticBindingContract()
+        {
+            Dictionary<string, Type> contract = new Dictionary<string, Type>(StringComparer.OrdinalIgnoreCase);
+            contract.Add("PartitionContext", typeof(PartitionContext));
+
+            // TODO: can we determine here if the binding is single dispatch?
```

Comment:
I'd really like to be able to determine single dispatch here - not sure how. If we can't, it won't be a huge issue - even if a customer tried to use these properties for an EventData[], it would fail at runtime.